### PR TITLE
Update pathology tumor detection bundle

### LINKED ISSUE #185: Missing files in the pathology bundle
**Describe the bug**
I read the README doc of Pathology bundle and it says something about  “configs/environment.json” and “prepare_inference_data.sh”:
https://github.com/Project-MONAI/model-zoo/tree/dev/models/pathology_tumor_detection#data-preparation
But actually I didn’t find these 2 files in the configs of the bundle.

### PR DESCRIPTION
**Pull request by bhashemian**
Fixes #185 https://github.com/Project-MONAI/MONAI/issues/5472

### Description
Update pathology tumor detection bundle to make it easier to switch between cucim and openslide, add FROC scripts, and update README.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [x] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [x] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [x] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/pathology_tumor_detection/configs/metadata.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
     "version": "0.3.3",
```
Comment (Nic-Ma) [inline models/pathology_tumor_detection/configs/metadata.json]: Your version is not updated to `0.4.0`.

Thanks.

File: models/pathology_tumor_detection/docs/README.md [hunk 2]
```diff
@@ -112,6 +113,7 @@ The model needs to be used with NVIDIA hardware and software. For hardware, the
 [1] He, Kaiming, et al, "Deep Residual Learning for Image Recognition." In Proceedings of the IEEE conference on computer vision and pattern recognition, pp. 770-778. 2016. <https://arxiv.org/pdf/1512.03385.pdf>
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/docs/README.md:103]: Hi @drbeh , should the `Intended Use` section and the license of clara be removed?

**Review by yiheng-wang-nv — APPROVED** [review]

**Pull request merged**